pkg/cloudmap/resource_manager.go

```diff
@@ -385,3 +390,32 @@ func (m *defaultResourceManager) isCloudMapServiceOwnedByVirtualNode(ctx context
 func (m *defaultResourceManager) buildCloudMapServiceSummaryCacheKey(nsSummary *servicediscovery.NamespaceSummary, serviceName string) string {
 	return fmt.Sprintf("%s/%s", awssdk.StringValue(nsSummary.Id), serviceName)
 }
+
+func (m *defaultResourceManager) getClusterNodeInfo(ctx context.Context) map[string]nodeAttributes {
+	var nodeInfoByName map[string]nodeAttributes
```

Comment:
don't define nodeInfoByName here if it's not initialized later. we can just return nil if list fails.